Detectors/TRD/reconstruction/src/CruRawReader.cxx

```diff
@@ -195,23 +279,29 @@ int CruRawReader::processHBFs()
 
     // copy the contents of the current RDH into the buffer to be parsed, RDH payload is memory size minus header size
     std::memcpy((char*)&mHBFPayload[0] + mTotalHBFPayLoad, ((char*)rdh) + headerSize, rdhpayload);
+    int packetCount = o2::raw::RDHUtils::getPacketCounter(rdh);
```

Comment:
is this variable used for anything?

Reply:
used to be, but check is somewhere else for consecutive packetcount.